pkg/util/validation.go

```diff
@@ -18,19 +18,34 @@ package util
 
 import (
 	"regexp"
-	"strings"
 )
 
-// IsDNSLabel tests for a string that conforms to the definition of a label in
-// DNS (RFC 1123).
-func IsDNSLabel(value string) bool {
-	return IsDNS1123Label(value)
+const LabelValueFmt string = "[A-Za-z0-9_\\-\\.,]*"
+
+var labelValueRegexp = regexp.MustCompile("^" + LabelValueFmt + "$")
+
+const LabelValueMaxLength int = 63
+
+func IsValidLabelValue(value string) bool {
+	return (len(value) <= LabelValueMaxLength && labelValueRegexp.MatchString(value))
 }
 
-// IsDNSSubdomain tests for a string that conforms to the definition of a
-// subdomain in DNS (RFC 1123).
-func IsDNSSubdomain(value string) bool {
-	return IsDNS1123Subdomain(value)
+const AnnotationValueFmt string = ".*"
```

Comment:
This seems like a waste of cycles to actually check, and a waste of LoC to retain - what if we just droped it all in favor of a comment "// annotation values are opaque" ?